packages/core/src/metrics/MetricsService.ts

```diff
@@ -63,7 +66,7 @@ export class MetricsService {
 
     const { publisher } = this;
 
-    if (!publisher) {
+    if (!publisher || !this.defaultMetrics) {
       console.info("Metrics server configuration is missing. Metrics will be disabled.");
       return Promise.resolve();
```

Comment:
Not for this PR but perhaps we should return that msg in the promise as `err` instead of printing in the console.

Reply:
I think if we check that before creating metrics then we will avoid having all of this :)

Reply:
The problem is `publish` is a public method so that the user can call it, hence we have to have this check.

Reply:
@josemigallas You are right about the method but user will not have access to metrics just other SDK's that use core.